specs/capella/validator.md

```diff
@@ -58,13 +58,6 @@ All validator responsibilities remain unchanged other than those noted below.
 expected withdrawals for the slot must be gathered from the `state` (utilizing the
 helper `get_expected_withdrawals`) and passed into the `ExecutionEngine` within `prepare_execution_payload`.
```

Comment:
This needs more clarification that it *must* be the post-process_slots state. This is stated in the Phase 0 document but this is a distinct "gotcha" with how we are doing withdrawals building here so is worth an explicit (re)callout